manifests/4.5/logforwardings.crd.yaml

```diff
@@ -29,6 +29,7 @@ spec:
                     enum:
                       - elasticsearch
                       - forward
+                      - syslog-legacy
```

Comment:
Nit here, but I don't think we need to call the type 'syslog-legacy'.  I think 'syslog' is sufficient regardless off the actual plugin.  From what I recall, any plugin to replace what we have would be a superset of what the current one provides.  Is that correct?

Reply:
Yes, that's correct - in terms of protocols, the new plugin should be a superset of what the current one provides.